src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY_INDEX = "supply";
+    private static final String BUY_INDEX = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(createStatistic(readFromFile(fromFileName)), toFileName);
+    }
+
+    private List<String> readFromFile(String fromFileName) {
+        File fromFile = new File(fromFileName);
+        List<String> fileData = new ArrayList<>();
+        try {
+            fileData = Files.readAllLines(fromFile.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't read file " + fromFileName, e);
```

lets make exception messages more informative